en/asgardeo/docs/references/actions/pre-update-password-action/api/pre_update_password_action-v1.yaml

```diff
@@ -0,0 +1,267 @@
+          type: string
+          description: The unique identifier for the user store.
+          example: UFJJTUFSWQ==
+        name:
+          type: string
+          description: "User store name used to identify the user store in configuration settings, user interfaces, and administrative tasks."
+          example: PRIMARY
+      description: Indicates the user store in which the user's data is being managed.
+    AdditionalData:
+      type: object
+      description: Defines the additional data related to the updating credential.
+      anyOf:
+      - $ref: '#/components/schemas/AdditionalDataForHashedPassword'
+    AdditionalDataForHashedPassword:
+      type: object
+      properties:
+        algorithm:
+          type: string
+          enum:
+          - SHA256
+      description: Defines additional data related to hashed passwords.
+    SuccessResponse:
+      type: object
+      properties:
+        actionStatus:
+          type: string
+          enum:
+          - SUCCESS
+      description: Defines the success response.
+    FailedResponse:
+      type: object
+      properties:
+        actionStatus:
+          type: string
+          enum:
+          - FAILED
+          description: Indicates the outcome of the request. For a failed operation, this should be set to FAILED.
+        failureReason:
+          type: string
+          description: Provides the reason for failing password update.
+        failureDescription:
+          type: string
+          description: Offers a detailed explanation of the failure
+    ErrorResponse:
+      type: object
+      properties:
+        actionStatus:
+          type: string
+          enum:
+          - ERROR
+          description: Indicates the outcome of the request. For an error operation, this should be set to ERROR.
+        errorMessage:
+          type: string
+          description: The cause of the error.
+        errorDescription:
+          type: string
+          description: A detailed description of the error.
+      description: |
+         When the external service responds with an ERROR state, it can return an HTTP status code of 400, 401, or 500, indicating either a validation failure or an issue processing the request. 
+    RequestBody:
+      type: object
+      properties:
+        actionType:
+          type: string
+          description: "Specifies the action being triggered, which in this case is PRE_UPDATE_PASSWORD."
+          enum:
+          - PRE_UPDATE_PASSWORD
+        event:
+          $ref: '#/components/schemas/Event'
+  securitySchemes:
+    BasicAuth:
+      type: http
+      scheme: basic
+    BearerAuth:
+      type: http
+      scheme: bearer
+    ApiKeyAuth:
+      type: apiKey
+      name: X-API-Key
+      in: header
+    OAuth2:
+      type: oauth2
+      flows:
+        clientCredentials:
+          tokenUrl: https://example.com/oauth/token
+          scopes:
+            process: process request generate response
+            
```

Comment:
Add a new line

Reply:
There's a new line added check line 267. Not sure why this still says no new line

Reply:
It's because L267 has some whitespace characters.